test/signalcorr.jl

```diff
@@ -9,88 +9,76 @@ using Test
 
 # random data for testing
 
-x = [-2.133252557240862    -.7445937365828654;
```

Comment:
We should still test the real case?

Reply:
Should we? It would amount to testing the real case in ```LinearAlgebra.dot()```

Reply:
Just to avoid changing code and test simultaneously.